docs/govuk-verify.md

```diff
@@ -35,53 +35,191 @@ curl localhost:50300/admin/healthcheck
-process detailed in the section "2. Submit certificate signing requests" in the
-private Verify docs entitled "GOV.UK Verify Certification Process for Relying
-Party Subscribers". You will be prompted for the challenge phrase you submitted
-during that process.
+Once complete, the csr, keys and pass-phrase are all added to your local
+directory in `tmp/certs` and uploaded to the Azure Keyvault.
 
-When it's finished, the generated keys, CSRs, and the challenge phrase will be
-automatically stored in the Key Vault on Azure matching the environment they
-were generated for.
+**Be mindful to delete these files once your are confident they have been
+enrolled (see below).**
 
-Integration certificates expire after 2 years, while production certificates
-expire after 6 months. See the
-[Verify docs for how to rotate keys](https://www.docs.verify.service.gov.uk/maintain-your-connection/rotate-keys/)
-for more information. `bin/request-vsp-certs` will probably be helpful.
+##### Enrol
+
+Complete these steps for both signing and encryption certificates.
+
+Open the following document:
+
+https://docs.google.com/document/d/1lH4b2uOn35rdzLAzW5I-_O9wG9NU8fTre8FaM0Y97L8/edit?pli=1#
+
+And go to the appropriate url for the environment you are enrolling the
+certificates for, either integration or production.
+
+- Choose ENROL
+- Upload the CSR file
+- Enter the details in the form:
+  - Applicant Details: your name and the group email address
+    (capt-dev@digital.education.gov.uk), ensue the email address is correct as
+    this is where the certificate will be sent.
+  - Certificate Profile: Select the type of certificate either SAML Signing or
+    SAML Encryption, ensure this matches the CSR you are uploading
+  - Challenge Phrase: the challenge phrase used above
+
+Once submitted you will receive a confirmation email, the certificates will be
+generated within 5 days and we be sent to the group email address.
 
-#### Importing certificates
+##### Import
 
-Once we get certificates back from Verify, we want to store them in the Key
-Vault alongside the keys and requests. Run the following to automate that
-process.
+When you receive the certificates run the import script for both signing and
+encryption certificates:
 
 ```bash
 bin/import-vsp-certs <environment> <version>
 ```
 
-Where `<environment>` and `<version>` match the values used to generate the
-certificate request above.
+Using the same environment and version as you set in the `request-vsp-certs`
+script.
+
+When prompted, paste the certificate body for the type of certificate (signing
+or encryption) - be careful to note which type of certificate you are importing.
+
+The script copies the certificates to the Azure Keyvault.
+
+##### Rotate
+
+The steps to actually rotate the keys varies between the two types of
+certificate
+
+[Verify docs for this step](https://www.docs.verify.service.gov.uk/maintain-your-connection/rotate-keys/#rotate-your-encryption-and-signing-keys)
+
+###### Signing
```

Comment:
```suggestion
###### Signing key
```

It think it would be good to be clear that this isn't an action "we need to sign something" and is a label of the type of key.